app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -463,7 +463,7 @@ private void updateContributionFragmentTabContent(boolean isContributionsListFra
 
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        Timber.d(data.toString());
+        Timber.d(data!=null?data.toString():"");
```

Comment:
Could you add a message saying that the intent data was empty instead of "" (assuming the relevance of the log over here).

Reply:
Yes something like "onActivityResult data is null" would be more useful when reading the log :-)